src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -1037,10 +1042,12 @@ private void SendHeartbeatStatusToSource(BlittableJsonTextWriter writer, long la
                 [nameof(ReplicationMessageReply.LastIndexTransformerEtagAccepted)] = lastIndexOrTransformerEtag,
                 [nameof(ReplicationMessageReply.Exception)] = null,
                 [nameof(ReplicationMessageReply.DocumentsChangeVector)] = documentChangeVectorAsDynamicJson,
-                [nameof(ReplicationMessageReply.IndexTransformerChangeVector)] = indexesChangeVectorAsDynamicJson
+                [nameof(ReplicationMessageReply.IndexTransformerChangeVector)] = indexesChangeVectorAsDynamicJson,
+                [nameof(ReplicationMessageReply.DbId)] = _database.DbId
             });
 
             writer.Flush();
+            LastHeartbeatTicks = DateTime.UtcNow.Ticks;
```

Comment:
if you have access to DocumentDatabase then use DocumentDatabase.Time.UtcNow, if not then SystemTime.UtcNow